bedrock-devnet/devnet/__init__.py

```diff
@@ -94,8 +94,13 @@ def main():
         devnet_l1_genesis(paths)
         return
 
-    log.info('Building docker images')
-    run_command(['docker', 'compose', 'build', '--progress', 'plain'], cwd=paths.ops_bedrock_dir, env={
+    git_commit = subprocess.run(['git', 'rev-parse', 'HEAD'], capture_output=True, text=True).stdout.strip()
+    git_date = subprocess.run(['git', 'show', '-s', "--format='%ct'"], capture_output=True, text=True).stdout.strip()
```

Comment:
```suggestion
    git_date = subprocess.run(['git', 'show', '-s', "--format=%ct"], capture_output=True, text=True).stdout.strip()
```

There's no shell interpreting things so the single quotes were being included in the content resulting in:
```
t=2023-10-23T23:34:51+0000 lvl=info msg="Initializing rollup node"            version=v0.0.0-03c0061b-'1698096523'
```